config/examples/backend_redis.yml

```diff
@@ -1,29 +1,18 @@
-# Configuration for Redis. Separate per each environment.
-#
-# Accepted options are:
-#
-# servers       list of servers (hostname:port) to connect to. The first one is the master, the
-#               others are used for failover (no sharding yet).
-# db            database to use. A number from 0 to 10 (default is 0)
-# timeout
-# password
-# thread_safe
 base: &base
   url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/0') %>"
+  timeout: "<%= ENV['BACKEND_REDIS_TIMEOUT'] || '1' %>"
```

Comment:
In openshift config this was set to `timeout: 1` explicitly, so adding an env var here and in the openshift config. But not sure, maybe there is a reason why we do not allow to override the value...